applications/nrf_desktop/README.rst

```diff
@@ -177,700 +177,705 @@ To simplify, the diagram shows only data related to HID input reports that are s
 
    nRF Desktop mouse HID data sensing and transmission
 
-.. _nrf_desktop_porting_guide:
+Requirements
+************
 
-Integrating your own hardware
-*****************************
+The project comes with configuration files for the following boards:
 
-This section describes how to adapt the nRF Desktop application to different hardware.
-It describes the configuration sources that are used for the default configuration, and lists steps required for adding a new board:
+    * nRF52840 DK (PCA10056) - the application is configured to work as a gaming mouse (motion emulated using DK buttons)
```

Comment:
Yes, this is something planned for the final review task: https://projecttools.nordicsemi.no/jira/browse/DESK-685
I won't be making these changes here to avoid too many edits not related to the task.